src/schema.rs

```diff
@@ -228,6 +247,13 @@ struct TeamPeople {
     include_all_team_members: bool,
 }
 
+#[derive(serde::Deserialize, Debug)]
+#[serde(rename_all = "kebab-case", deny_unknown_fields)]
+struct GitHubData {
+    name: Option<String>,
```

Comment:
Could this struct be something like `GithubTeam` or at least a comment added on `name` saying that this is the team name?

Reply:
Maybe it's best to rename it:

```toml
[github]
team-name = "release-alt"
orgs = ["rust-lang"]
```

Reply:
Yes, that works too!

Reply:
Renamed it to `team-name`.